### PR TITLE
Add duration device class for total print time and longest print sensors

### PR DESCRIPTION
**Pull request by saintonotole**
This patch adds a device class to the `Longest Print` and `Totals Print Time` history sensors, similar to the rest of the existing time-related sensors.
The current implementation casts these sensors to plain text values, which makes it very difficult to try to build any automations on top of these values.
For example, I want to make a reminder every X hours of printing to do maintenance on the printer (lubricate the guides, check belt tension, check for debris in the extruder, and so on).
This could also be useful for people with print farms to more easily keep track of printer wear and tear and to more evenly distribute print jobs.

### CODE REVIEW

**Comment by marcolivierarsenault** [comment]
@saintonotole this is a great patch, can you adjust tests for it

**Comment by saintonotole** [comment]
@marcolivierarsenault I've adjusted the tests to match the expected data

File: custom_components/moonraker/sensor.py [hunk 1]
```diff
@@ -573,11 +575,13 @@ async def async_setup_history_sensors(coordinator, entry, async_add_entities):
         MoonrakerSensorDescription(
             key="longest_print",
             name="Longest Print",
-            value_fn=lambda sensor: convert_time(
-                sensor.coordinator.data["history"]["job_totals"]["longest_print"]
-            ),
+            value_fn=lambda sensor: sensor.coordinator.data["history"]["job_totals"][
```
Comment (marcolivierarsenault) [inline custom_components/moonraker/sensor.py:578]: For the failing test you probably need to round the value

**Comment by marcolivierarsenault** [comment]
inactive, re-open if needed

**Pull request closed without merging**